__init__.py

```diff
@@ -174,3 +174,17 @@ def __init__(self, opsdroid, config):
         time = message.regex.group('time')
         instant_recovery = rubrik.vsphere_instant_recovery(vm_name, date, time)
         await message.respond('All done! {} has been recovered to {} at {}. Response: {}'.format(vm_name, date, time, instant_recovery))
+
+    @match_regex('get current live mounts of vmware VM (?P<vm_name>[\w\'-]+)')
+    async def getvspherelivemounts(self, message):
+        """
+        A skills function to list the current live mounts of a VMware VM. The parser looks for the message argument.
+
+        Arguments:
+            message {str} -- get current live mounts of vmware VM {vm_name}
+        """
+        rubrik = rubrik_cdm.Connect()
+        vm_name = message.regex.group('vm_name')
+        live_mount = rubrik.get_vsphere_live_mount(vm_name)
+        live_mount_names = rubrik.get_vsphere_live_mount_names(vm_name)
+        await message.respond('All done! {} has the following live mounts: {}. Response: {}'.format(vm_name, live_mount_names,live_mount))
```

Comment:
Potential the response from get_vsphere_live_mount will muddy up the chat session with the json response. Consider splitting into two separate functions to get names and get details